__tests__/app.test.js

```diff
@@ -193,3 +193,77 @@ describe("GET /reviews/:review_id/comments", () => {
       });
   });
 });
+
+describe("POST /api/reviews/:review_id/comments", () => {
+  test("POST - status:201, inserts a new comment into the database and responds with newly added comment", () => {

```

Comment:
Add another 201 test for unnecessary properties - e.g. send a comment with username; body; unnecessary_key; and it returns the comment without the unnecessary_key.